components/eventing-controller/pkg/errors/errors_internal_unit_test.go

```diff
@@ -0,0 +1,25 @@
+package errors
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var errInvalidStorageType = NewArgumentError("invalid stream storage type: %q")
+
+func Test_ArgumentError_WithArg(t *testing.T) {
+	t.Parallel()
+	// given
+	errorWithoutArg := errInvalidStorageType
+	someArgument := "some argument"
+
+	// when
+	errorWithArg := errInvalidStorageType.WithArg(someArgument)
+
+	// then
+	// ensure errors are still equal, otherwise Is() would return false.
+	assert.Equal(t, errorWithoutArg.errorFormatType, errorWithArg.errorFormatType)
+	// ensure argument is set
```

Comment:
```suggestion
	// Ensure errors are still equal, otherwise Is() would return false.
	assert.Equal(t, errorWithoutArg.errorFormatType, errorWithArg.errorFormatType)
	// Ensure argument is set.
```

Reply:
nitty:
If it has a fullstop, it is a sentence. If it is a sentence, start with a capital letter. :)